intuita-webview/src/jobDiffView/App.tsx

```diff
@@ -97,6 +105,19 @@ function App() {
 		return null;
 	}
 
+	if (view.metadataMarkdown !== null) {
```

Comment:
I think that you should pass `kind` within the `viewProps` to signify the kind of the view